build.gradle

```diff
@@ -57,18 +57,19 @@ allprojects {
     }
   }
 
-  configurations.all {
-    resolutionStrategy {
-      force 'com.fasterxml.jackson:jackson-bom:2.13.4'
-      force 'com.fasterxml.jackson.core:jackson-databind:2.13.4'
-      force 'org.apache.orc:orc-core:1.8.3'
-      force 'com.google.guava:guava:31.1-jre'
+  if (it.path != ':integrations:spark-3.5:openhouse-spark-3.5-itest') {
```

Comment:
this force version is causing runtime error in spark apps:
> Caused by: com.fasterxml.jackson.databind.JsonMappingException: Scala module 2.15.2 requires Jackson Databind version >= 2.15.0 and < 2.16.0 - Found jackson-databind version 2.13.4

due to spark3.5 change log:
> Upgrade FasterXML jackson to 2.15.2 [SPARK-42354](https://issues.apache.org/jira/browse/SPARK-42354) [SPARK-43774](https://issues.apache.org/jira/browse/SPARK-43774) [SPARK-43904](https://issues.apache.org/jira/browse/SPARK-43904)

and in general a global configuration force like this should be avoided.

Reply:
Yes. Jackson 2.13.4 does not work with spark 3.5. That's why I added this line to avoid the version forcing on spark 3.5 modules. We have not upgraded the apps to 3.5 yet, so not included the `apps` module here.
I don't know the historical reason we use version force here, but it is worth trying removing it.